Update WebExtension API processId (#23958)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/webnavigation/onreferencefragmentupdated/index.md

  - : `integer`. The ID of the tab in which the navigation is about to occur.
- `url`
  - : `string`. The URL to which the given frame will navigate.
- `processId` {{optional_inline}} {{deprecated_inline}}

>   - : `integer`. This value is never set in modern browsers. It used to represent the ID of the process running the renderer for this tab.

Reviewer: ```suggestion
  - : `integer`. This value is not set in modern browsers. When it was set, it represented the ID of the process running the renderer for this tab.
```
